compiler/rustc_error_codes/src/error_codes/E0321.md

```diff
@@ -1,11 +1,9 @@
-A cross-crate opt-out trait was implemented on something which wasn't a struct
+A cross-crate auto trait was implemented on something which wasn't a struct
 or enum type.
 
 Erroneous code example:
 
 ```compile_fail,E0321
-#![feature(auto_traits)]
```

Comment:
should this use negative_impls?

Reply:
I guess I can add it but it doesn't seem mandatory. Apparently, stderr is allowed to contain more errors than listed after `compile_fail`. Several other error code code blocks also lack `#![feature]`.

Reply:
yeah, it doesn't seem mandatory but it's nice to have it

Reply:
Addressed